Add support for custom annotation validation schema (#951)
RoadieHQ/roadie-backstage-plugins: .changeset/late-cobras-chew.md

---
'@roadiehq/roadie-backstage-entity-validator': minor
---

> Add option to provide url for fetching custom annotation schema for validation

Reviewer: ```suggestion
Add option to specify location of the schema file.
```
